lightning/src/routing/router.rs

```diff
@@ -3838,16 +3837,19 @@ mod tests {
 		}
 	}
 
+	#[cfg(not(feature = "no_std"))]
 	pub(super) fn random_init_seed() -> u64 {
 		// Because the default HashMap in std pulls OS randomness, we can use it as a (bad) RNG.
 		use core::hash::{BuildHasher, Hasher};
 		let seed = std::collections::hash_map::RandomState::new().build_hasher().finish();
 		println!("Using seed of {}", seed);
 		seed
 	}
+    #[cfg(not(feature = "no_std"))]
```

Comment:
nit: indentation with tabs.

Reply:
If there's an easy way to just swap spaces for tabs, sure (I don't think there is, though?), but we've tried rustfmt a few times and it makes tons of code completely unreadable. It's desire to lay out all your code completely vertically means you lose a ton of context when reviewing more often than not. There's a few issues open upstream to make it more sensible, but I don't believe there's been much progress on any of them.

Reply:
Ok, I won't worry about it then. Thought there might be a simple fix.